src/components/pages/rules/rules.js

```diff
@@ -1,14 +1,32 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 import React, { Component } from 'react';
-
+import RulesGrid from './rulesGrid/rulesGrid';
+import { Btn, RefreshBar } from 'components/shared';
 import './rules.css';
 
 export class Rules extends Component {
+  constructor(props) {
+    super(props);
+
+    if (!this.props.rules.length) {
+      this.props.fetchRules();
```

Comment:
Need to call isFunc?

Reply:
Not really, at least not here. The propTypes should do that but I haven't done that yet since the props haven't been finalized.